Dockerfile

```diff
@@ -18,8 +18,8 @@ RUN apt-get update && \
       libffi-dev \
       libtool \
       openssl \
-      python-dev \
-      python-pip \
+      python3-dev \
```

Comment:
Please rebase against latest `master`. Note that we are now using `python:3.8-slim-buster` base image (due to deps), so you have to keep `python-pip` here.